lib/matplotlib/image.py

```diff
@@ -1190,6 +1189,19 @@ def set_cmap(self, cmap):
             raise RuntimeError('Cannot change colors after loading data')
         super().set_cmap(cmap)
 
+    def get_cursor_data(self, event):
+        # docstring inherited
+        x, y = event.xdata, event.ydata
+        if (x < self._Ax[0] or x > self._Ax[-1] or
+                y < self._Ay[0] or y > self._Ay[-1]):
+            return None
```

Comment:
We should be able to test out-of-bounds in the new test as well, I think?

Reply:
👍 added